src/main/java/objectsBasicsMandatoryPracticalTasks/Clock/Clock.java

```diff
@@ -0,0 +1,137 @@
+package objectsBasicsMandatoryPracticalTasks.Clock;
+
+class Clock {
+    private long hours;
+    private long minutes;
+    private long seconds;
+    private static final int SEC_IN_DAYS = 86400;
+    private static final int SEC_IN_HOURS = 3600;
+    private static final int SEC_IN_MINUTE = 60;
+
+    Clock() {
+        this.hours = 12;
+        this.minutes = 0;
+        this.seconds = 0;
+    }
+
+    Clock(long hours, long minutes, long seconds) {
+        this.hours = hours;
+        this.minutes = minutes;
+        this.seconds = seconds;
+    }
+
+    Clock(int secondsSinceMidnight) {
+        Clock clock = new Clock();
```

Comment:
You've created new Clock in costructor which has nothing common with current Clock object (`this` object).
So that you didn't initialize `hours`, `minutes`, `seconds` in `this` object and if I call `getHours` for example, I get `0`